### DISCUSSION TITLE
master是有状态的，但是使用helm部署，master没有指定具体的node，如何数据的稳定性

### DISCUSSION THREAD

**Discussion by GounGG** [cubefs/cubefs#3441] [Q&A]
3 master节点的集群，指定了node label(component.cubefs.io/master: enabled)，如果我当前同时重启2个节点的pod,导致pod节点的位置对调了，会发生什么？

或者我有个节点要下线，无法拷贝master的data文件，这个时候在一台新的node上启动master加入到集群，会自动同步数据吗？

**Comment by leonrayang** [comment]
@mingwei
